gelfclient/client.py

```diff
@@ -5,6 +5,12 @@
 import struct 
 from datetime import datetime
 
+class LogLevel(IntEnum):
+    alert = 1
+    warning = 2
+    error = 3
```

Comment:
Sorry, I think I wrote a comment here before but forgot to submit it properly.

There's no need to put these values here when they are already defined in the standard library in the syslog module:
https://docs.python.org/2/library/syslog.html
> Priority levels (high to low):
LOG_EMERG, LOG_ALERT, LOG_CRIT, LOG_ERR, LOG_WARNING, LOG_NOTICE, LOG_INFO, LOG_DEBUG.